source/reference/mongosync.txt

```diff
@@ -182,6 +193,27 @@ show an empty ``system.views`` collection in that database. The empty
 ``system.views`` collection will not change the accuracy of user
 data on the destination.
 
+Write Blocking
+~~~~~~~~~~~~~~
+
+By default, write-blocking is not enabled during synchronization. When
+write-blocking is enabled, ``mongosync`` blocks writes:
+
+- On the destination cluster during replication
+- On the source cluster while committing
+
+To enable write-blocking, start ``mongosync`` with
+``enableUserWriteBlocking`` set to ``true``.
```

Comment:
I think this should link to the `/start` API as `enableUserWriteBlocking` an option there.

Reply:
> start mongosync with enableUserWriteBlocking set to true.

I still think this is misleading to the end users. It is not clear if this means start of the mongosync process or start of the replication. Since this is a /start API, we meant start of the replication.